src/NodaTime.Test/Calendars/YearMonthDayCalculatorTest.cs

```diff
@@ -13,37 +13,52 @@ namespace NodaTime.Test.Calendars
     [TestFixture]
     public class YearMonthDayCalculatorTest
     {
+        // Here the term "Islamic" only refers to whether the implementation is IslamicYearMonthDayCalculator,
+        // not whether the calendar itself is based on Islamic scripture.
+        // TODO(code review): Make sure I've reported a bug for the CodeRush Roslyn preview test runner - it
```

Comment:
Not yet. Going to write up a small test example. Now that I have an internet connection, I can email myself to keep it in my inbox until I've done it though - will remove the comment :)